### PR TITLE
feat: improve formatting Jinja blocks in attributes

### PR DESCRIPTION
**Pull request by g-plane**
Please note that interpolations in attribute name and attribute value won't be formatted. This is expected since they're parsed as plain texts.

cc @UnknownPlatypus

### CODE REVIEW

**Comment by UnknownPlatypus** [comment]
> Please note that interpolations in attribute name and attribute value won't be formatted. This is expected since they're parsed as plain texts.

I think it's better like that for now. Parsing and formatting interpolation in attribute value will add complexity and is not always safe if whitespace are important (in `href` values for example)

File: markup_fmt/tests/fmt/jinja/attributes/tag-or-block-as-attr.snap [hunk 1]
```diff
@@ -59,15 +67,18 @@ source: markup_fmt/tests/fmt.rs
...
       data-should_delay
     {% endif %}
   {% else %}
-    data-start='08:30:00' data-end='20:30:00' data-start_day='{% now "Y-m-d" %}'
-    data-saturday_start='08:30:00' data-saturday_end='20:30:00'
+    data-start="08:30:00"
+    data-end="20:30:00"
+    data-start_day='{% now "Y-m-d" %}'
+    data-saturday_start="08:30:00"
+    data-saturday_end="20:30:00"
```
Comment (UnknownPlatypus) [inline markup_fmt/tests/fmt/jinja/attributes/tag-or-block-as-attr.snap:81]: This is great :partying_face:

File: markup_fmt/tests/fmt/jinja/attributes/tag-or-block-as-attr.snap [hunk 2]
```diff
@@ -88,21 +101,26 @@ source: markup_fmt/tests/fmt.rs
...
   {% endif %}
 >
 
 <div
   class="flex foo"
   id="index_id"
-  {% if index_type == "foo" %}style="display: none"{% endif %}
+  {% if index_type == "foo" %}
+    style="display: none"
+  {% endif %}
```
[the remark this answers is not in the document]
Comment (g-plane) [reply markup_fmt/tests/fmt/jinja/attributes/tag-or-block-as-attr.snap:114]: Initially I tried it but I found that will be ugly.

**Comment by UnknownPlatypus** [comment]
@g-plane This seems to cause an issue for the following snippet that is not parsing anymore: 

```html
<picture
    {% for key, val in attributes.items %} 
        {{ key }}="{{ val }}"
    {% endfor %}
></picture>
```

I get this `Failed to parse "src/t.html" with error Syntax(SyntaxError { kind: ExpectJinjaTag, pos: 61, line: 3, column: 11 })`. This is failing to parse `{{ key }}` because it expect a JinjaTag.

**Pull request merged**